### PR TITLE
iOS dfu

### PR DESCRIPTION
**Pull request by Looveh**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	"dependencies": {
 		"react": "16.0.0-alpha.12",
 		"react-native": "0.45.1",
-		"react-native-ble-manager": "^5.1.1",
+		"react-native-ble-manager": "git://github.com/Looveh/react-native-ble-manager/#expose-ios-central-manager",
```
Comment (vikeri) [inline example/package.json]: Is this how it is supposed to be?

File: index.js [hunk 2]
```diff
@@ -1,13 +1,6 @@
...
 const { RNNordicDfu } = NativeModules;
-const NordicDFU = {
-  startDFU: Platform.select({
-    android: startDFU,
-    ios: () => {
-      console.warn("iOS not yet supported");
-    }
-  })
-};
+const NordicDFU = { startDFU };
```
Comment (vikeri) [inline index.js]: Did you run `prettier` on this?

Comment (Looveh) [inline index.js]: Yes

File: example/package.json (hunk 1 above)
Comment (vikeri) [inline example/package.json]: 👍

File: index.js (hunk 2 above)
Comment (vikeri) [inline index.js]: 👍

**Pull request merged**